geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml

```diff
@@ -20,6 +20,7 @@
                 <exclude>org.locationtech.geomesa:geomesa-geojson*</exclude>
                 <exclude>org.locationtech.geomesa:geomesa-blobstore*</exclude>
                 <exclude>org.locationtech.geomesa:geomesa-accumulo-compute*</exclude>
+                <exclude>org.locationtech.geomesa:geomesa-accumulo-spark-runtime*</exclude>
```

Comment:
it's still in `dist/spark/`, just not under `lib/` - i think the docs are ok as is?